snapcraft/cli/_errors.py

```diff
@@ -68,20 +68,25 @@ def exception_handler(exception_type, exception, exception_traceback, *,
     exit_code = 1
     is_snapcraft_error = issubclass(exception_type, errors.SnapcraftError)
     is_raven_setup = RavenClient is not None
+    is_sentry_enabled = os.getenv('SNAPCRAFT_ENABLE_SENTRY') is not None
+    is_sentry_flag = os.getenv('SNAPCRAFT_SEND_ERROR_DATA', 'n') == 'y'
```

Comment:
I think using `distutils.util.strtobool` here will lead to more intuitive results for setting both of these variables:

```
>>> distutils.util.strtobool('y')
1

>>> distutils.util.strtobool('N')
0

>>> distutils.util.strtobool('on')
1

>>> distutils.util.strtobool('off')
0
```

This even bit me in testing. We use this elsewhere to good effect, too.